pulsar-client/src/main/java/org/apache/pulsar/client/impl/BinaryProtoLookupService.java

```diff
@@ -84,6 +92,21 @@ public BinaryProtoLookupService(PulsarClientImpl client,
         this.serviceNameResolver = new PulsarServiceNameResolver();
         this.listenerName = listenerName;
         updateServiceUrl(serviceUrl);
+
+        Attributes attrs = Attributes.of(AttributeKey.stringKey("transport-type"), "binary");
+
+        histoGetBroker = client.instrumentProvider().newLatencyHistogram("pulsar.client.lookup",
```

Comment:
Since the unit is `seconds`, the metric name gets change (at least in Prometheus exporter) to: 

```
# HELP pulsar_client_lookup_seconds Duration of lookup operations
# TYPE pulsar_client_lookup_seconds histogram
pulsar_client_lookup_seconds_bucket{otel_scope_name="org.apache.pulsar.client",otel_scope_version="3.3.0-SNAPSHOT",pulsar_lookup_transport_type="binary",pulsar_lookup_type="metadata",success="true",le="5.0E-4"} 0
```

Reply:
Prometheus exporter implements their own rules conforming to OpenMetrics format I think. If you ship directly to DataDog using OTLP, they may convert with their own rules. You can also ship using OTLP to the collector and from there the Promeus converter there will do its job.
I think it's important in OTel we keep the naming as they have prescribed.